students/21th/jeongeonwoo/js/main.js

```diff
@@ -0,0 +1,54 @@
+const mainRight = document.querySelector('#main-right');
+const inputComment = document.querySelectorAll('.article-bottom-input-commnet');
+const inputButton = document.querySelectorAll('.article-bottom-input-posting');
+const commentText = document.querySelectorAll('.article-bottom-commnet.last');
+
+//mainRight fixed move left
+mainRight.style.left = `${167 + (window.innerWidth / 2)}px`;
+
+window.addEventListener('resize', () => {
+    mainRight.style.left = `${167 + (window.innerWidth / 2)}px`;
+})
+
+//inputcomment click button undisabled
+for (let i = 0; i < inputComment.length; i++) {
+    inputComment[i].addEventListener('keyup', () => {
+        if (inputComment[i].value) {
+            inputButton[i].disabled = false;
+        } else inputButton[i].disabled = true;
```

Comment:
```suggestion
        inputButton[i].disabled = !inputComment[i].value;
```
if 문에 사용된 조건문 자체가 boolean임을 이용하여 위와 같이 사용할 수 있어보입니다.

Reply:
변경했습니다 감사합니다!